### PR TITLE
Use Khan/actions

### PR DESCRIPTION
**Pull request by jaredly**
Summary:

Issue: https://khanacademy.atlassian.net/browse/FEI-4581

## Test plan:
See the checks on this PR

### CODE REVIEW

**Review by somewhatabstract — APPROVED** [review]
LGTM. Thanks for tidying this up  :)

File: .github/workflows/node-ci.yml [hunk 1]
```diff
@@ -112,6 +112,10 @@ jobs:
         # Build production
         build-script: "build:prodsizecheck"
 
+      - uses: Khan/actions@check-for-changeset-v0
+        with:
+          exclude: .github/,.storybook/
```
[the remark this answers is not in the document]
Comment (somewhatabstract) [reply .github/workflows/node-ci.yml:117]: I presume this doesn't prevent dependabot updates?

[the remark this answers is not in the document]
Comment (jaredly) [reply .github/workflows/node-ci.yml:117]: oh that's a good question. I can exclude dependabot

File: .github/workflows/node-ci.yml [hunk 2]
```diff
@@ -128,6 +112,11 @@ jobs:
         # Build production
         build-script: "build:prodsizecheck"
 
+      - uses: Khan/actions@check-for-changeset-v0
```
Comment (somewhatabstract) [inline .github/workflows/node-ci.yml:115]: What does this do for the Version Packages PR that changeset creates? Does it just ignore it?

Comment (jaredly) [reply .github/workflows/node-ci.yml:115]: yeah, the check-for-changeset action checks if the branch name is `changeset-release/main` and ignores it.

**Pull request merged**